src/show.cpp

```diff
@@ -83,11 +86,11 @@ void Show::printProperty(const char* title, const char* name,
     {
         std::string val;
         std::visit(
-            [&val](auto&& arg) {
+            [&val, &boolVals](auto&& arg) {
                 using T = std::decay_t<decltype(arg)>;
                 if constexpr (std::is_same_v<T, bool>)
                 {
-                    val = arg ? "Enabled" : "Disabled";
+                    val = arg ? boolVals.second : boolVals.first;
```

Comment:
I'm wondering why `first` and `second` are swapped?
It seems to me that it would be more logic when `first` is a positive value and `second` is negative.